deployment_scripts/deploy_linux.sh

```diff
@@ -1,10 +1,43 @@
 #!/bin/bash
-source config
 
 exists() {
   command -v "$1" >/dev/null 2>&1
 }
 
+is_root() {
+  return $(id -u)
+}
+
+has_sudo() {
+  # 0 true, 1 false
+  timeout 1 sudo id && return 0 || return 1
+}
+
+handle_error() {
+  echo "Fix the errors above and rerun the script"
+  exit 1
+}
+
+log_message() {
+  echo -e "\n\n"
+  echo -e "DEPLOYMENT SCRIPT: $1"
+}
+
+config_branch=${2:-"develop"}
+config_url="https://raw.githubusercontent.com/guardicore/monkey/${config_branch}/deployment_scripts/config"
+
+if exists curl; then
+  file=$(mktemp)
+  curl -s -o $file "$config_url"
```

Comment:
So if user doesn't have curl it's done? use `wget -q $config_url -O $file || curl $config_url -k -o $file` and if it fails then present error

Reply:
That's going to be a fun commit for you to review :))))